build.gradle

```diff
@@ -146,6 +146,17 @@ allprojects {
     mavenCentral()
   }
 
+  def checkJavaVersion = { task ->
+    if(JavaVersion.current() != JavaVersion.VERSION_11){
+       throw new GradleException("Project must be build with version 11")
+    }
+  }
+
+  afterEvaluate {
+    for(def task in it.tasks.build)
```

Comment:
What tasks are included under `it.tasks.build`... does it correspond to all dependencies of `build` task? 🤔

Reply:
it.tasks.build correspond to each module build task. So it doesn't matter if we trigger the build task from whole plugin or from another Gradle module inside them -> checkJavaVersion will be call